test/App/DocsSpec.hs

```diff
@@ -65,7 +64,6 @@ urlsToCheck =
   , rEnvLockFileGenerateDocUrl
   , refPodDocUrl
   , refGradleDocUrl
-  , containerScanningDocUrl
```

Comment:
TODO: Update with new documentation link once https://github.com/fossas/fossa-cli/pull/1077 is merged/rebased.